dbt_subprojects/dex/models/aggregator_trades/dex_aggregator_trades.sql

```diff
@@ -44,8 +43,8 @@ SELECT
     , token_pair
     , token_bought_amount
     , token_sold_amount
-    , token_bought_amount_raw
-    , token_sold_amount_raw
+    , try_cast(token_bought_amount_raw as int256) as token_bought_amount_raw
+    , try_cast(token_sold_amount_raw as int256) as token_sold_amount_raw
```

Comment:
this adds a try_cast() for handling invalid int256 values, i.e. 115792089237316195423570985008687907853269984665640564039457584007913129639935

Reply:
can you help me understand why we want to output `int256`, rather than `uint256`?

here is what i looked at.
```
select
  *
from
  dex_aggregator.trades
order by
  token_bought_amount_raw desc
limit
  100
  
```

grab a tx from output that contains that massive raw amount value.
https://arbiscan.io/tx/0x45988ee7c3da3a5ab1c034221e972d7e6ead3cede9e73b7387cdb67b5d95b1a7#eventlog#5

if i hover over the value, it says `uint256` in the block explorer. can we not write it out that way too?

Reply:
yes that's a good point - the try_cast(() remain should remain `uint256`

without the try_cast(), it throws an overflow error when trying to cast that large value as `uint256`